rush.json

```diff
@@ -50,6 +50,12 @@
       "packageName": "@bentley/geo-tools-react",
       "projectFolder": "packages/geo-tools",
       "versionPolicyName": "geo-tools"
+    },
+    {
+      "packageName": "@bentley/measure-tools-react",
+      "projectFolder": "packages/measure-tools",
+      "versionPolicyName": "measure-tools",
+      "shouldPublish": false

```

Comment:
@calebmshafer are you cool with this? Or should we wait until tomorrow to decide what to do with this pkg?

Reply:
this works with me. Lets get this merged in and then we can figure out what else to do.